packages/dev/loaders/src/SPLAT/splatFileLoader.ts

```diff
@@ -76,15 +79,21 @@ export class SPLATFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPlu
 
     /**
      * Creates loader for gaussian splatting files
+     * @param loadingOptions options for loading and parsing splat and PLY files.
      */
-    constructor() {}
+    constructor(loadingOptions?: Partial<Readonly<SPLATLoadingOptions>>) {
+        this._loadingOptions = loadingOptions || SPLATFileLoader._DefaultLoadingOptions;
+    }

```

Comment:
You can also just do this, correct?
```suggestion
    constructor(loadingOptions: Partial<Readonly<SPLATLoadingOptions>> = SPLATFileLoader._DefaultLoadingOptions) {
        this._loadingOptions = loadingOptions;
    }

```